src/client/eppo-client.spec.ts

```diff
@@ -281,9 +304,29 @@ describe('EppoClient E2E test', () => {
     expect(assignment).toEqual('control');
   });
 
-  function getAssignments(subjects: string[], experiment: string): string[] {
+  function getAssignments(
+    subjects: string[],
+    experiment: string,
+    valueTestType: ValueTestType = ValueTestType.StringType,
+  ): (EppoValue | null)[] {
     return subjects.map((subjectKey) => {
-      return globalClient.getAssignment(subjectKey, experiment);
+      switch (valueTestType) {
+        case ValueTestType.BoolType: {
+          const ba = globalClient.getBoolAssignment(subjectKey, experiment);
+          if (ba === null) return null;
+          return EppoValue.Bool(ba);
```

Comment:
minor but with short-circuiting and I think we could simplify this to `return ba && EppoValue.Bool(ba)`.

```node
> v = null
null
> v && Bool(v)
null
> v = true
true
> v && Boolean(v)
true
```